pkg/controller/volume/persistentvolume/metrics/metrics.go

```diff
@@ -56,6 +73,8 @@ type PVCLister interface {
 func Register(pvLister PVLister, pvcLister PVCLister) {
 	registerMetrics.Do(func() {
 		prometheus.MustRegister(newPVAndPVCCountCollector(pvLister, pvcLister))
+		prometheus.MustRegister(VolumeBindingAddToSchedulerBinderCache)
```

Comment:
I think the registration here is not going to work.  This registration is called in PV controller process, but the volume binding cache is a library used by scheduler process.  So I think the library needs export a register method that the scheduler will call.

Reply:
These registration calls should be removed from this file, because this is called for PV controller process, not scheduler process.